apis/core/v1beta1/featureflagsource_types.go

```diff
@@ -17,10 +17,41 @@ limitations under the License.
 package v1beta1
 
 import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/open-feature/open-feature-operator/common/utils"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	SidecarEnvVarPrefix              string = "SIDECAR_ENV_VAR_PREFIX"
+	InputConfigurationEnvVarPrefix   string = "SIDECAR"
+	SidecarMetricPortEnvVar          string = "METRICS_PORT"
+	SidecarPortEnvVar                string = "PORT"
+	SidecarSocketPathEnvVar          string = "SOCKET_PATH"
+	SidecarEvaluatorEnvVar           string = "EVALUATOR"
+	SidecarImageEnvVar               string = "IMAGE"
+	SidecarVersionEnvVar             string = "TAG"
+	SidecarProviderArgsEnvVar        string = "PROVIDER_ARGS"
+	SidecarDefaultSyncProviderEnvVar string = "SYNC_PROVIDER"
+	SidecarLogFormatEnvVar           string = "LOG_FORMAT"
+	SidecarProbesEnabledVar          string = "PROBES_ENABLED"
+	defaultSidecarEnvVarPrefix       string = "FLAGD"
+	DefaultMetricPort                int32  = 8014
+	defaultPort                      int32  = 8013
+	defaultSocketPath                string = ""
+	defaultEvaluator                 string = "json"
+	defaultImage                     string = "ghcr.io/open-feature/flagd"
+	// renovate: datasource=github-tags depName=open-feature/flagd/flagd
+	defaultTag           string = "v0.6.3"
```

Comment:
Please update to the compatible latest flagd version - [v0.7.0](https://github.com/open-feature/flagd/releases/tag/flagd%2Fv0.7.0)

Reply:
Yep, @odubajDT a compatible flagd version was released: https://github.com/open-feature/flagd/pkgs/container/flagd/148036560?tag=v0.7.0

Reply:
good catch, somehow missed this piece :)